src/javascript/JContent/actions/deletePermanentlyAction.jsx

```diff
@@ -7,7 +7,8 @@ import {PATH_CONTENTS_ITSELF, PATH_FILES_ITSELF} from './actions.constants';
 
 const checkAction = node => node.operationsSupport.markForDeletion &&
     isMarkedForDeletion(node) &&
-    node.aggregatedPublicationInfo.publicationStatus === 'NOT_PUBLISHED';
+    node.aggregatedPublicationInfo.publicationStatus === 'NOT_PUBLISHED' &&
+    node.aggregatedPublicationInfo.existsInLive !== undefined && !node.aggregatedPublicationInfo.existsInLive;
```

Comment:
I don't think this works as expected. As soon as existsInLive is defined then the whole expression will always resolve to false, right?

The green part should be something like:
`publicationStatus === 'NOT_PUBLISHED' && (existsInLive === undefined) ? true : !existsInLive`

I've omitted the 'node.aggregatedPublicationInfo' parts for the sake of clarification here.

Reply:
you're right we should keep the old behaviour